### ISSUE TITLE
Bug: Regression - Opentsdb aggregators list not coming from API

### ISSUE THREAD

**Issue by utkarshcmu** [grafana/grafana#4056]
In 2.6 Grafana version, the list of Opentsdb aggregators are coming from Opentsb /api/aggregators api, which is the required behavior. But in the master branch currently, the list of aggregators are coming from the hard coded values in opentsdb/query_ctrl.js

I am not sure why this is happening into master branch now, I tested querying my opentsdb setup and /api/aggregators api which is giving me the following list of aggregators:
```
["mimmin","ep95r3","p999","count","dev","ep95r7","sum","ep90r3","ep90r7","avg","min","ep50r7","max","mimmax","p99","ep99r7","ep999r7","ep50r3","ep999r3","ep75r3","p90","ep75r7","p50","p75","p95","ep99r3","zimsum"]
```

But Opentsdb queryCtrl shows this list which is hardcoded in query_ctrl.js:
```
['avg', 'sum', 'min', 'max', 'dev', 'zimsum', 'mimmin', 'mimmax']
```

**Comment by utkarshcmu** [comment]
So, the last commit which broke this functionality is https://github.com/grafana/grafana/commit/0b25a29a9c389decc7637be3e4f9fd958dd917b3

Before this commit, the list of OpenTSDB aggregators were coming via API.

**Comment by utkarshcmu** [comment]
this bug should be fixed before it starts breaking user's dashboards.

**Comment by utkarshcmu** [comment]
@bergquist Can you please have a quick look into this issue? It might be a quick fix for you as I am not a pro in Angular. The main problem is "this.aggregators" is not getting updated when the "datasource.getAggregators()" returns the list of aggregators from the Opentsdb datasource.

**Comment by utkarshcmu** [comment]
Fixed by #4161
